e2e/scripts/smoke_test.sh

```diff
@@ -71,14 +72,22 @@ rook_infra::gather_results() {
 }
 
 rook_infra::cleanup() {
-    echo Removing rook-test-framework container...
-    docker kill ${id}
-    docker rm ${id}
-}
+    local tag_name=$1
+    local rook_platform=$2
+    local k8s_version=$3
 
-{ #try
+    #Run clean up tests that runs down on dind script
+    docker exec -t ${id} /bin/bash -c \
+        "go test -timeout 1200s -run TestRookInfraCleanUp  ${git_test_directory} --rook_platform=${rook_platform} --k8s_version=${k8s_version} --rook_version=${tag_name} -v 2>&1"
 
-    if [ -z "$1" ]; then
+    echo Removing rook-test-framework container and images...
+    docker kill ${id} || true
+    docker rm ${id} || true
+    docker rmi $(docker images|grep 'rook-test\|kubeadm-dind-cluster\|ceph/base') > /dev/null 2>&1 || true
```

Comment:
does this grep only output a single image? if multiple images it's likely going to fail. You might need to pipe the results of the grep to `xargs docker rmi`